packages/venia-concept/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+    "logo.svg": "node_modules/new_logo/logo.svg"
```

Comment:
So the plugin literally just swaps the resource with another? Interesting.

Could you extend this to a component/module rather than an asset? Something that takes props would be interesting. The API would need to be rock solid. Imagine replacing the entirety of checkout!

Reply:
That is exactly what I am working on right now. Trying to replace the `Toast` component using an `npm` module that takes the same props and just changes the way UI looks.

But the ultimate plan is to create an environment for HTML extensions similar to the ones that exist in the current marketplace.